src/test/java/seedu/address/testutil/TypicalStudent.java

```diff
@@ -46,17 +48,20 @@
             .withEmail("anna@example.com").build();
 
     // Manually added
```

Comment:
perhaps this should start with lowercase?
```suggestion 
    // manually added
```